tasks_list.go

```diff
@@ -220,18 +232,19 @@ type DiscoveryNode struct {
 
 // TaskInfo represents information about a currently running task.
 type TaskInfo struct {
-	Node               string      `json:"node"`
-	Id                 int64       `json:"id"` // the task id (yes, this is a long in the Java source)
-	Type               string      `json:"type"`
-	Action             string      `json:"action"`
-	Status             interface{} `json:"status"`      // has separate implementations of Task.Status in Java for reindexing, replication, and "RawTaskStatus"
-	Description        interface{} `json:"description"` // same as Status
-	StartTime          string      `json:"start_time"`
-	StartTimeInMillis  int64       `json:"start_time_in_millis"`
-	RunningTime        string      `json:"running_time"`
-	RunningTimeInNanos int64       `json:"running_time_in_nanos"`
-	Cancellable        bool        `json:"cancellable"`
-	ParentTaskId       string      `json:"parent_task_id"` // like "YxJnVYjwSBm_AUbzddTajQ:12356"
+	Node               string                 `json:"node"`
+	Id                 int64                  `json:"id"` // the task id (yes, this is a long in the Java source)
+	Type               string                 `json:"type"`
+	Action             string                 `json:"action"`
+	Status             interface{}            `json:"status"`      // has separate implementations of Task.Status in Java for reindexing, replication, and "RawTaskStatus"
+	Description        interface{}            `json:"description"` // same as Status
+	StartTime          string                 `json:"start_time"`
+	StartTimeInMillis  int64                  `json:"start_time_in_millis"`
+	RunningTime        string                 `json:"running_time"`
+	RunningTimeInNanos int64                  `json:"running_time_in_nanos"`
+	Cancellable        bool                   `json:"cancellable"`
+	ParentTaskId       string                 `json:"parent_task_id"` // like "YxJnVYjwSBm_AUbzddTajQ:12356"
+	Headers            map[string]interface{} `json:"headers"`
```

Comment:
This could be a `http.Headers` as well, if [`TaskInfo.java`](https://github.com/elastic/elasticsearch/blob/6.x/server/src/main/java/org/elasticsearch/tasks/TaskInfo.java#L209-L213) is up-to-date.

If you don't have the time to correct this, don't worry. I will merge this probably over the weekend.